lib/policies/proxy/round-robin.js

```diff
@@ -0,0 +1,30 @@
+const logger = require('../../logger').gateway;
+
+module.exports = class RoundRobin {
+  constructor (proxy, endpoints) {
```

Comment:
Hmm.  Don't know.  Just felt right to delegate proxying responsibility to the load balancer, so it can make whatever decisions it needs to or return an error of some kind if it must without bouncing it back to the caller (proxy.js in this case).